src/schemas/__init__.py

```
@@ -0,0 +1,19 @@
+from schemas.movies import (
+    MovieDetailSchema,
```

Ruff (0.11.9)

`schemas.movies.MovieDetailSchema` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)